Open5GS daemon v2.4.4

08/31 10:58:45.664: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
08/31 10:58:45.665: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf_cld.log' (../lib/app/ogs-init.c:132)
08/31 10:58:45.693: [sock] ERROR: socket bind(2) [192.168.0.112]:8805 failed (99:Cannot assign requested address) (../lib/core/ogs-socket.c:109)
08/31 10:58:45.693: [sock] ERROR: udp_server() [192.168.0.112]:8805 failed (99:Cannot assign requested address) (../lib/core/ogs-udp.c:62)
08/31 10:58:45.693: [app] ERROR: Failed to intialize UPF (../src/upf/app.c:28)
08/31 10:58:45.693: [app] FATAL: Open5GS initialization failed. Aborted (../src/main.c:219)
Open5GS daemon v2.4.4

08/31 11:05:19.801: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
08/31 11:05:19.801: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf_cld.log' (../lib/app/ogs-init.c:132)
08/31 11:05:19.829: [pfcp] INFO: pfcp_server() [192.168.0.112]:8805 (../lib/pfcp/path.c:30)
08/31 11:05:19.830: [gtp] INFO: gtp_server() [192.168.0.112]:2152 (../lib/gtp/path.c:30)
08/31 11:05:19.830: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
08/31 11:05:20.362: [pfcp] INFO: ogs_pfcp_connect() [192.168.0.111]:8805 (../lib/pfcp/path.c:59)
08/31 11:05:20.362: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
08/31 12:32:22.602: [app] INFO: SIGHUP received (../src/main.c:58)
Open5GS daemon v2.4.4

08/31 12:32:50.696: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
08/31 12:32:50.696: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf_cld.log' (../lib/app/ogs-init.c:132)
08/31 12:32:50.702: [pfcp] INFO: pfcp_server() [192.168.0.112]:8805 (../lib/pfcp/path.c:30)
08/31 12:32:50.702: [gtp] INFO: gtp_server() [192.168.0.112]:2152 (../lib/gtp/path.c:30)
08/31 12:32:50.702: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
08/31 12:32:51.284: [pfcp] INFO: ogs_pfcp_connect() [192.168.0.111]:8805 (../lib/pfcp/path.c:59)
08/31 12:32:51.284: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
Open5GS daemon v2.4.4

08/31 14:54:05.154: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
08/31 14:54:05.154: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf_cld.log' (../lib/app/ogs-init.c:132)
08/31 14:54:05.181: [pfcp] INFO: pfcp_server() [192.168.0.112]:8805 (../lib/pfcp/path.c:30)
08/31 14:54:05.181: [gtp] INFO: gtp_server() [192.168.0.112]:2152 (../lib/gtp/path.c:30)
08/31 14:54:05.182: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
08/31 14:54:05.533: [pfcp] INFO: ogs_pfcp_connect() [192.168.0.111]:8805 (../lib/pfcp/path.c:59)
08/31 14:54:05.534: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
08/31 14:54:17.910: [upf] INFO: [Added] Number of UPF-Sessions is now 1 (../src/upf/context.c:160)
08/31 14:54:17.911: [gtp] INFO: gtp_connect() [127.0.0.4]:2152 (../lib/gtp/path.c:58)
08/31 14:54:17.911: [upf] INFO: UE F-SEID[CP:0x1 UP:0x1] APN[internet] PDN-Type[1] IPv4[10.45.0.2] IPv6[] (../src/upf/context.c:361)
08/31 14:54:18.079: [gtp] INFO: gtp_connect() [192.168.0.131]:2152 (../lib/gtp/path.c:58)
Open5GS daemon v2.4.4

08/31 17:08:38.106: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
08/31 17:08:38.106: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf_cld.log' (../lib/app/ogs-init.c:132)
08/31 17:08:38.129: [pfcp] INFO: pfcp_server() [192.168.0.112]:8805 (../lib/pfcp/path.c:30)
08/31 17:08:38.130: [gtp] INFO: gtp_server() [192.168.0.112]:2152 (../lib/gtp/path.c:30)
08/31 17:08:38.131: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
08/31 17:08:38.720: [pfcp] INFO: ogs_pfcp_connect() [192.168.0.111]:8805 (../lib/pfcp/path.c:59)
08/31 17:08:38.721: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
08/31 17:08:50.809: [upf] INFO: [Added] Number of UPF-Sessions is now 1 (../src/upf/context.c:160)
08/31 17:08:50.809: [gtp] INFO: gtp_connect() [127.0.0.4]:2152 (../lib/gtp/path.c:58)
08/31 17:08:50.811: [upf] INFO: UE F-SEID[CP:0x1 UP:0x1] APN[internet] PDN-Type[1] IPv4[10.45.0.2] IPv6[] (../src/upf/context.c:361)
08/31 17:08:50.960: [gtp] INFO: gtp_connect() [192.168.0.131]:2152 (../lib/gtp/path.c:58)
Open5GS daemon v2.4.4

09/04 08:29:53.456: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
09/04 08:29:53.456: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf_cld.log' (../lib/app/ogs-init.c:132)
09/04 08:29:53.483: [pfcp] INFO: pfcp_server() [192.168.0.112]:8805 (../lib/pfcp/path.c:30)
09/04 08:29:53.483: [gtp] INFO: gtp_server() [192.168.0.112]:2152 (../lib/gtp/path.c:30)
09/04 08:29:53.484: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
09/04 08:37:47.996: [app] INFO: SIGHUP received (../src/main.c:58)
Open5GS daemon v2.4.4

09/18 09:20:51.116: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
09/18 09:20:51.116: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf_cld.log' (../lib/app/ogs-init.c:132)
09/18 09:20:51.146: [pfcp] INFO: pfcp_server() [192.168.0.112]:8805 (../lib/pfcp/path.c:30)
09/18 09:20:51.146: [gtp] INFO: gtp_server() [192.168.0.112]:2152 (../lib/gtp/path.c:30)
09/18 09:20:51.147: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
09/18 09:20:52.691: [pfcp] INFO: ogs_pfcp_connect() [192.168.0.111]:8805 (../lib/pfcp/path.c:59)
09/18 09:20:52.691: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
Open5GS daemon v2.4.4

09/18 09:26:03.124: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
09/18 09:26:03.124: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf_cld.log' (../lib/app/ogs-init.c:132)
09/18 09:26:03.135: [pfcp] INFO: pfcp_server() [192.168.0.112]:8805 (../lib/pfcp/path.c:30)
09/18 09:26:03.135: [gtp] INFO: gtp_server() [192.168.0.112]:2152 (../lib/gtp/path.c:30)
09/18 09:26:03.135: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
09/18 09:26:03.771: [pfcp] INFO: ogs_pfcp_connect() [192.168.0.111]:8805 (../lib/pfcp/path.c:59)
09/18 09:26:03.771: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
09/18 09:26:15.800: [upf] INFO: [Added] Number of UPF-Sessions is now 1 (../src/upf/context.c:160)
09/18 09:26:15.800: [gtp] INFO: gtp_connect() [127.0.0.4]:2152 (../lib/gtp/path.c:58)
09/18 09:26:15.801: [upf] INFO: UE F-SEID[CP:0x1 UP:0x1] APN[internet] PDN-Type[1] IPv4[10.45.0.2] IPv6[] (../src/upf/context.c:361)
09/18 09:26:15.945: [gtp] INFO: gtp_connect() [192.168.0.131]:2152 (../lib/gtp/path.c:58)
09/18 09:40:34.223: [pfcp] ERROR: ogs_pfcp_sendto() failed (101:Network is unreachable) (../lib/pfcp/path.c:115)
09/18 09:40:34.223: [pfcp] ERROR: ogs_pfcp_xact_commit: Expectation `rv == OGS_OK' failed. (../lib/pfcp/xact.c:573)
09/18 09:40:34.393: [app] INFO: SIGHUP received (../src/main.c:58)
Open5GS daemon v2.4.4

09/18 09:41:07.715: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
09/18 09:41:07.715: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf_cld.log' (../lib/app/ogs-init.c:132)
09/18 09:41:07.738: [pfcp] INFO: pfcp_server() [192.168.0.112]:8805 (../lib/pfcp/path.c:30)
09/18 09:41:07.738: [gtp] INFO: gtp_server() [192.168.0.112]:2152 (../lib/gtp/path.c:30)
09/18 09:41:07.739: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
09/18 09:41:08.294: [pfcp] INFO: ogs_pfcp_connect() [192.168.0.111]:8805 (../lib/pfcp/path.c:59)
09/18 09:41:08.295: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
09/18 09:41:20.276: [upf] INFO: [Added] Number of UPF-Sessions is now 1 (../src/upf/context.c:160)
09/18 09:41:20.276: [gtp] INFO: gtp_connect() [127.0.0.4]:2152 (../lib/gtp/path.c:58)
09/18 09:41:20.276: [upf] INFO: UE F-SEID[CP:0x1 UP:0x1] APN[internet] PDN-Type[1] IPv4[10.45.0.2] IPv6[] (../src/upf/context.c:361)
09/18 09:41:20.421: [gtp] INFO: gtp_connect() [192.168.0.131]:2152 (../lib/gtp/path.c:58)
Open5GS daemon v2.4.4

09/22 15:17:32.611: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
09/22 15:17:32.611: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf_cld.log' (../lib/app/ogs-init.c:132)
09/22 15:17:32.624: [pfcp] INFO: pfcp_server() [192.168.0.112]:8805 (../lib/pfcp/path.c:30)
09/22 15:17:32.624: [gtp] INFO: gtp_server() [192.168.0.112]:2152 (../lib/gtp/path.c:30)
09/22 15:17:32.624: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
09/22 15:17:34.477: [pfcp] INFO: ogs_pfcp_connect() [192.168.0.111]:8805 (../lib/pfcp/path.c:59)
09/22 15:17:34.478: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
09/22 15:17:48.067: [upf] INFO: [Added] Number of UPF-Sessions is now 1 (../src/upf/context.c:160)
09/22 15:17:48.067: [gtp] INFO: gtp_connect() [127.0.0.4]:2152 (../lib/gtp/path.c:58)
09/22 15:17:48.067: [upf] INFO: UE F-SEID[CP:0x1 UP:0x1] APN[internet] PDN-Type[1] IPv4[10.45.0.2] IPv6[] (../src/upf/context.c:361)
09/22 15:17:48.214: [gtp] INFO: gtp_connect() [192.168.0.131]:2152 (../lib/gtp/path.c:58)
Open5GS daemon v2.4.4

09/22 16:29:09.008: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
09/22 16:29:09.008: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf_cld.log' (../lib/app/ogs-init.c:132)
09/22 16:29:09.015: [pfcp] INFO: pfcp_server() [192.168.0.112]:8805 (../lib/pfcp/path.c:30)
09/22 16:29:09.015: [gtp] INFO: gtp_server() [192.168.0.112]:2152 (../lib/gtp/path.c:30)
09/22 16:29:09.016: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
09/22 16:29:09.678: [pfcp] INFO: ogs_pfcp_connect() [192.168.0.111]:8805 (../lib/pfcp/path.c:59)
09/22 16:29:09.679: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
09/22 16:29:21.354: [upf] INFO: [Added] Number of UPF-Sessions is now 1 (../src/upf/context.c:160)
09/22 16:29:21.354: [gtp] INFO: gtp_connect() [127.0.0.4]:2152 (../lib/gtp/path.c:58)
09/22 16:29:21.354: [upf] INFO: UE F-SEID[CP:0x1 UP:0x1] APN[internet] PDN-Type[1] IPv4[10.45.0.2] IPv6[] (../src/upf/context.c:361)
09/22 16:29:21.504: [gtp] INFO: gtp_connect() [192.168.0.131]:2152 (../lib/gtp/path.c:58)
09/22 16:45:32.629: [upf] INFO: [Added] Number of UPF-Sessions is now 2 (../src/upf/context.c:160)
09/22 16:45:32.629: [upf] INFO: UE F-SEID[CP:0x2 UP:0x3] APN[internet] PDN-Type[1] IPv4[10.45.0.3] IPv6[] (../src/upf/context.c:361)
